recipes/dependency-walker/LICENSE

```diff
@@ -0,0 +1,3 @@
+from http://www.dependencywalker.com/:
+
+Dependency Walker is completely free to use. However, you may not profit from the distribution of it, nor may you bundle it with another product.
```

Comment:
Hmm it isn't clear to me that we aren't "bundling it with another product."  We may need to get permision from them or ask the NF lawyers